payloads/external/edk2/Makefile

```diff
@@ -345,6 +345,22 @@ endif
 ifeq ($(CONFIG_EDK2_SERIAL_ON_SUPERIO),y)
 BUILD_STR += -D UART_ON_SUPERIO=TRUE
 endif
+# PcdShowCpuMenu = FALSE
+ifeq ($(CONFIG_EDK2_DASHARO_CPU_CONFIG),y)
+BUILD_STR += --pcd gDasharoSystemFeaturesTokenSpaceGuid.PcdShowCpuMenu=TRUE
+endif
+# PcdShowCpuCoreDisable = FALSE
+ifeq ($(CONFIG_EDK2_CORE_DISABLE_OPTION),y)
+BUILD_STR += --pcd gDasharoSystemFeaturesTokenSpaceGuid.PcdShowCpuCoreDisable=TRUE
+endif
+# PcdShowCpuHyperThreading = FALSE
+ifeq ($(CONFIG_EDK2_HYPERTHREADING_OPTION),y)
+BUILD_STR += --pcd gDasharoSystemFeaturesTokenSpaceGuid.PcdShowCpuHyperThreading=TRUE
+endif
+# PcdCpuHyperThreadingDefault = TRUE
+ifneq ($(CONFIG_EDK2_HYPERTHREADING_DEFAULT_STATE),y)
```

Comment:
This breaks build for boards which use EDK without this option.  Need to either handle this option differently or update other boards with EDK to a new revision which could be done like:

```
sed -Ei '/Dasharo\/edk2(\.git)?"/,+1s/="[0-9a-fA-F]{40}"/="a7b33b10e9d86bf68e6e8cb89cd910fb6bc370c9"/g' configs/config.*
```

Reply:
Thanks, applied

Reply:
236a12e978f80764ae9f86c5f8a1ae859ae81726